mods/farming/api.lua

```diff
@@ -184,8 +145,11 @@ farming.register_plant = function(name, def)
 	for k, v in pairs(def.fertility) do
 		g[v] = 1
 	end
+
+	local mname, pname = unpack(name:split(":"))
+
 	minetest.register_node(":" .. mname .. ":seed_" .. pname, {
-		description = def.description,
+		description = def.description or "Seed",
```

Comment:
This is a bad idea, as it makes less useful descriptions of items - may just require description, and fail gracefully:

```Lua
if not def.description then
	error("Description needed in farming definition!")
end
```